diff --git a/src/lib/libraries/extensions/index.jsx b/src/lib/libraries/extensions/index.jsx
index ba18b91..e316f38 100644
--- a/src/lib/libraries/extensions/index.jsx
+++ b/src/lib/libraries/extensions/index.jsx
@@ -1,6 +1,9 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import radioInsetIconURL from './yourextension/your-extension-icon.png';
+import radioIconURL from './yourextension/your-extension-background.png';
+
 import musicIconURL from './music/music.png';
 import musicInsetIconURL from './music/music-small.svg';
 
@@ -47,6 +50,15 @@ import gdxforConnectionIconURL from './gdxfor/gdxfor-illustration.svg';
 import gdxforConnectionSmallIconURL from './gdxfor/gdxfor-small.svg';
 
 export default [
+    {
+        name: 'micro:bit Radio',
+        extensionId: 'yourScratchExtension',
+        iconURL: radioIconURL,
+        insetIconURL: radioInsetIconURL,
+        description: 'Use the micro:bit\'s Radio blocks within Scratch.',
+        featured: true,
+        disabled: false
+    },
     {
         name: (
             <FormattedMessage
